openlibrary/templates/site/body.html

```diff
@@ -31,9 +31,9 @@
       </div>
       $# Announcement banner will only be rendered if announcement and storage_key variables are set.
       $# Be sure to escape any single quotes inside of the announcement HTML string.
-      $ announcement = ''
+      $ announcement = 'We’re fighting to restore access to 500,000+ books in court this week. <a class="btn primary" href="https://blog.archive.org/2024/06/26/were-fighting-for-library-rights-in-court-this-friday-join-us/">Join us</a>'
```

Comment:
```suggestion
      $ announcement = 'We’re fighting to restore access to 500,000+ books in court this week. <a class="btn primary" href="https://blog.archive.org/2024/06/26/were-fighting-for-library-rights-in-court-this-friday-join-us/">Join us</a>'
```
```suggestion
      $ announcement = 'We’re fighting to restore access to 500,000+ books in court this week. <a class="btn primary" data-ol-link-track="BannerClick|FightForRightsBlog" href="https://blog.archive.org/2024/06/26/were-fighting-for-library-rights-in-court-this-friday-join-us/">Join us</a>'
```
Add analytics for blog link clicks.